google-datacatalog-rdbms-connector/src/google/datacatalog_connectors/rdbms/prepare/datacatalog_entry_factory.py

```diff
@@ -170,6 +170,17 @@ def __convert_source_system_timestamp_fields(raw_create_time,
 
     @staticmethod
     def __format_entry_column_type(source_name):
+        if isinstance(source_name, bytes):
+            # Use UTF-8 default encoding observed in MySQL, other RDBMS'
```

Comment:
Suggestion to make this comment a bit clearer:
```
            # We've noticed some MySQL instances use bytes-like objects
            # instead of `str` to specify the column types. We are using UTF-8
            # to decode such objects when it happens because UTF-8 is the
            # default character set for MySQL 8.0 onwards.
            #
            # We didn't notice similar behavior with other RDBMS but, if so, we
            # should handle encoding as a configuration option that each RDBMS
            # connector would have to set up. It might be exposed as a CLI arg,
            # so users could easily change that. There is also the option to
            # scrape that config directly from the DB.
```